producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
+
+This extension provides support for sending Change Data Capture(CDC) events to an Apache Kafka®.
+It is designed to work with the DataStax Stargate.
```

Comment:
"_DataStax_ Stargate"? I though we wanted to avoid explicit referenced to DataStax in Stargate

Reply:
Yes, remove that